code/modules/mob/living/carbon/xenomorph/Evolution.dm

```diff
@@ -139,12 +139,21 @@
 	else
 		new_xeno.plasma_stored = new_xeno.plasma_max*(plasma_stored/plasma_max) //preserve the ratio of plasma
 
+	for(var/i in src.built_structures)
+		new_xeno.built_structures += built_structures[i]
```

Comment:
when i said 'can't you just do' i meant remove this code and do the copy instead